README.md

```diff
@@ -55,9 +55,6 @@ executables for MacOS and only some for Linux.
     >
     > Execute: roslaunch central_control full_system.launch
     > 
-    > There is a chance of getting this message in the ROS console: WARNING:tornado.general:Connect error on fd 1372: WSAECONNREFUSED.
-      To fix this, search the code for "airsim.CarClient(ip=host_ip)" and replace it with "airsim.CarClient()" .
```

Comment:
Do we no longer have to add "ip=host_ip" inside the brackets?

Reply:
It shouldn't matter much as AirSim is likely to always run on the local machine, but keeping `ip=host_ip` gives us greater flexibility to change the address of the AirSim instance we want to control in the future.

Reply:
Just to add, if you don't specify `ip=host_ip`, Python attempts to connect to AirSim at `127.0.0.1`. `host_ip` is equal to this value by default, but it can easily be changed if we need to in the future.